src/main/java/teammates/ui/webapi/action/GetOngoingSessionsAction.java

```diff
@@ -45,24 +46,29 @@ public ActionResult execute() {
         long startTime;
         try {
             startTime = Long.parseLong(startTimeString);
-        } catch (NumberFormatException e) {
+            //test for bounds
+            Instant.ofEpochMilli(startTime).minus(30, ChronoUnit.DAYS).toEpochMilli();
```

Comment:
I believe you got the number `30` from the method `FeedbackSessionsDb#getAllOngoingSessions`, but now these two places can go out of sync. Let's extract this number to `Const`.